.github/workflows/ci-nuxtjs.yml

```diff
@@ -119,13 +119,20 @@ jobs:
   #     - name: Run System Tests
   #       shell: bash
   #       run: yarn test:system --filter=nuxt-app
-  
+
+  #     - uses: actions/upload-artifact@v3
+  #       if: always()
+  #       with:
+  #         name: nuxt-playwright-report
+  #         path: playwright-reports/nuxt-playwright-report/
+  #         retention-days: 7
+
   #     - name: Run Percy Tests
-  #       shell: bash
-  #       run: yarn test:visual --filter=nuxt-app
-  #   env:
-  #     BROWSERSTACK_USERNAME: ${{ secrets.BROWSERSTACK_USERNAME }}
-  #     BROWSERSTACK_ACCESS_KEY: ${{ secrets.BROWSERSTACK_ACCESS_KEY }}
-  #     NUXT_AMPLIFY_ID: d36dan3bxjue8c
-  #     PERCY_TOKEN_PIE_APERTURE_NUXT: ${{ secrets.PERCY_TOKEN_PIE_APERTURE_NUXT }}
-  #     PR_NUMBER: ${{ github.event.number }}
+  #     - shell: bash
+  #     - run: yarn test:visual --filter=nuxt-app
+    env:
```

Comment:
It's not obvious what these apply to, if it's the deploy Nuxt app step is it worth moving them up above the commented out steps?

Reply:
fixed!